### ISSUE TITLE
&lt; is not interpreted as less than sign

### ISSUE THREAD

**Issue by ahmedelmahalawey** [senny/sablon#79]
Hi,
I am trying to add text that contains a less than sign and has special styling, so I am using html converter to apply this and when I convert the less than character to &lt; it doesn't get rendered in the generated file although the greater than sign is rendered the same way by replcing it to &gt; in the HTML text.

Is there a way to apply this

Thanks

**Comment by stadelmanma** [comment]
@ahmedelmahalawey  so after doing some digging the reason we lose HTML entities is that we pass the provided HTML string though `Nokogiri::HTML.fragment` here: https://github.com/senny/sablon/blob/2d2399e7343618f82a63f2335ddd144023e85dc9/lib/sablon/html/converter.rb#L18 and then through `Nokogiri::XML.fragment` here: https://github.com/senny/sablon/blob/2d2399e7343618f82a63f2335ddd144023e85dc9/lib/sablon/content.rb#L75

The HTML entities defined initially get converted by Nokogiri during the HTML parsing step and then get lost in the translation to XML. [This stack overflow](https://stackoverflow.com/a/7759975) answer might be a possible solution but I'll need to investigate/test more.

I ran into this issue as well but since I was rendering the HTML from Rails partials I figured it was a Rails produced problem and never looked into it.
